rs/ic_os/config_types/src/lib.rs

```diff
@@ -108,10 +108,15 @@ pub struct SetupOSSettings;
 pub struct HostOSSettings {
     pub vm_memory: u32,
     pub vm_cpu: String,
+    #[serde(default = "default_vm_nr_of_vcpus")]
```

Comment:
I don’t think we should go too deep on this PR, but later it might be nice to `Default` the fields through types, so that any code users get the right values too. (We don’t really have any right now, so it doesn’t make a difference).